workflows/taxprofiler.nf

```diff
@@ -254,7 +256,10 @@ workflow TAXPROFILER {
     ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
     ch_multiqc_files = ch_multiqc_files.mix(ch_methods_description.collectFile(name: 'methods_description_mqc.yaml'))
     ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
-    ch_multiqc_files = ch_multiqc_files.mix(FASTQC.out.zip.collect{it[1]}.ifEmpty([]))
+
+    if (!params.perform_fastqc_alternative) {
+        ch_multiqc_files = ch_multiqc_files.mix(FASTQC.out.zip.collect{it[1]}.ifEmpty([]))
+    }
```

Comment:
missing FALCO mixing in?

Reply:
Right, I will update!

Reply:
I will have to look into this as I get the following error:

`
Access to 'FALCO.out' is undefined since process doesn't declare any output`

Reply:
You might be missing the channel name so I guess `.txt`

Reply:
```
    if (!params.preprocessing_qc_tool == 'falco') {
        ch_multiqc_files = ch_multiqc_files.mix(FASTQC.out.zip.collect{it[1]}.ifEmpty([]))
    }
    else {
        ch_multiqc_files = ch_multiqc_files.mix(FALCO.out.txt.collect{it[1]}.ifEmpty([]))
    }
```

I hope I do not miss something, but this is how I am mixing FALCO in.